crypto/Makefile

```diff
@@ -8,13 +8,13 @@ ADX_SUPPORT := $(shell if ([ -f "/proc/cpuinfo" ] && grep -q -e '^flags.*\badx\b
 .PHONY: test
 test:
 	# test all packages with Relic library enabled
-	GO111MODULE=on go test -coverprofile=$(COVER_PROFILE) $(if $(JSON_OUTPUT),-json,) --tags relic -v ./...
+	GO111MODULE=on go test -coverprofile=$(COVER_PROFILE) $(if $(TEST_LONG),,-short) $(if $(JSON_OUTPUT),-json,) --tags relic -v ./...
 cross-blst-test:
 	# test all packages with Relic library enabled
 ifeq ($(ADX_SUPPORT), 1)
-	GO111MODULE=on go test -coverprofile=$(COVER_PROFILE) $(if $(JSON_OUTPUT),-json,) --tags relic,blst -run BLST -v ./...
+	GO111MODULE=on go test -coverprofile=$(COVER_PROFILE) $(if $(TEST_LONG),,-short) $(if $(JSON_OUTPUT),-json,) --tags relic,blst -run BLST -v ./...
```

Comment:
I don't get why we need to specify the `-short` argument.

Is a long running test can automatically be detected, that's why you use `-short` to specify only run short test case?

What happen if we don't add the `-short` here, and check `TEST_LONG` env var before running the test which we consider is long running test. Just like what we do with other env vars you added.

Reply:
Yeah, originally added this because -short is a command line flag built into the `go` command.

However, I am gonna change this and just use env var for consistency.